operations/pyroscope/helm/pyroscope/rendered/micro-services.yaml

```diff
@@ -56,7 +56,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: "minio-sa"
-  namespace: "default"
+  namespace: "pyroscope"
```

Comment:
I think we should use the `default` namespace by default

Reply:
Where is this specified ? It gets changed when I do make helm/check